datadog/plugins/init.sls

```diff
@@ -0,0 +1,20 @@
+{% from "datadog/map.jinja" import datadog with context %}
+
+include:
+  - ..service
+
+datadog_plugin_configuration_directory:
+  file.directory:
+    - name: /etc/dd-agent/conf.d
+
+{% for name, settings in salt.pillar.get('datadog:integrations', {}).items() %}
```

Comment:
No, good catch. I had originally had the map file merging in the `datadog:lookups` key path which is why I had gone with the pillar approach at first. Now that it's just merging the whole datadog key path I could go with the map data. That being said, it's an open question as to whether it's a good idea to merge in the whole keyspace to the map file or if it should just be a subset of that data. Maybe make the convention be `<formula_name>:overrides` instead of `<formula_name>:lookups`. Thoughts?